src/controllers/organizationController.ts

```diff
@@ -414,6 +383,86 @@ export default class OrganizationController {
     return none;
   }
 
+  private createOnboardingDocuments(
+    ipaCode: string,
+    requestsCreationResponseSuccess: IResponseSuccessCreation<RequestCollection>
+  ): TaskEither<
+    IResponseErrorInternal,
+    IResponseSuccessCreation<RequestCollection>
+  > {
+    // TODO:
+    //  the documents must be stored on cloud (Azure Blob Storage).
+    //  @see https://www.pivotaltracker.com/story/show/169644958
+    const outputFolder = `./documents/${ipaCode}`;
+    const createDocumentPerRequest = (request: Request) => {
+      const rejectError = (maybeError: Option<Error>) => {
+        if (isSome(maybeError)) {
+          return Promise.reject(maybeError.value);
```

Comment:
never reject, return `Either<Error, ...>` instead (or the left side of a TaskEither)